src/types.ts

```diff
@@ -110,10 +111,10 @@ export interface StoredParseResult {
 interface SubActionBase<U extends string, T> extends DataInterval<T> {
     type: U;
 }
-
 export type SubAction =
     | SubActionBase<"hover", string>
-    | SubActionBase<"format", string>;
+    | SubActionBase<"format", string>
+    | SubActionBase<"highlight", HighlightScope>;
```

Comment:
Would you mind alphabetising these (I know they weren't alphabetical previously)